collection.go

```diff
@@ -88,6 +88,43 @@ func (c *LoggerCollection) getLogger(name string, level LogLevel,
 	return logger
 }
 
+// ConfigureLoggers configures loggers according to the given string
+// specification, which specifies a set of loggers and their associated
+// logging levels.  Loggers are semicolon-separated; each
+// configuration is specified as <logger>=<level>.  White space outside of
+// logger names and levels is ignored.  The default level is specified
+// with the name "DEFAULT".
+//
+// An example specification:
+//	`<root>=ERROR; foo.bar=WARNING`
```

Comment:
doesn't look like `<root>` is actually handled?

Reply:
Removing that and suggesting use of -log.level for specifying the default.